packages/localizations/src/zh-CN.ts

```diff
@@ -196,7 +196,7 @@ export const zhCN: LocalizationResource = {
         subtitle: '允许用户根据已验证的电子邮件域名自动加入组织或请求加入。',
         title: '已验证域名',
       },
-      successMessage: '组织已更新。',
+      successMessage: '组织信息已更新。',
```

Comment:
@youyou-sudo Let's keep this the same. According to Google Translate, the existing message specifically calls out organization which is more ideal.

Reply:
Thank you for your feedback! As my first PR, I am glad to receive your guidance.
Also, I have added a Changeset. Could you please take a look? Thank you!

Reply:
@youyou-sudo This is great, thank you!